bin/ns

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env ruby
```

Comment:
What about logs of these scripts (`bin/ns`, `bin/ns_unshared`)? Do we need them for debugging? Currently, it presents only in the console.

Reply:
It is unlikely that they will be needed for debugging HLK tests. `bin/ns` and `bin/ns_unshared` only concerns about initialization, cleanup, and forwarding between the outer namespace (for Internet and operator access) and the inner namespace.
The network constructs the guest will operate on are entirely contained in a network namespace so we don't have to concern about `bin/ns` and `bin/ns_unshared` once the namespace is constructed.

If you need to debug a setup issue (e.g. you tried `bin/ns` on a new system setup but it has namespace disabled), you may need the logs, but the console output should be enough in the case.